### PR TITLE
Update common-osd-overview.rst

### PR DESCRIPTION
**Pull request by vierfuffzig**
fix link reference from optional hardware toctree

### CODE REVIEW

**Comment by rmackay9** [comment]
Merged, thanks!

**Comment by vierfuffzig** [comment]
gosh that was fast. thanks a lot randy.

**Comment by rmackay9** [comment]
@vierfuffzig, yup, np.  I was hoping that fixing this would make the page appear in the optional hardware section but i'm still not seeing it..

File: common/source/docs/common-osd-overview.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. common-osd-overwiew:
+.. _common-osd-overwiew:
```
Comment (hamishwillee) [inline common/source/docs/common-osd-overview.rst]: You might want to fix spelling.

`.. _common-osd-overwiew:` to `.. _common-osd-overview:`

**Comment by hamishwillee** [comment]
Did you add the page to the `toctree` section of the proposed parent document - ie https://github.com/ArduPilot/ardupilot_wiki/blob/master/common/source/docs/common-optional-hardware.rst ?

This is what adds things to the sidebar.

File: common/source/docs/common-osd-overview.rst (hunk 1 above)
Comment (vierfuffzig) [reply common/source/docs/common-osd-overview.rst]: thanks a lot! checked that line a hundred times without noticing.

**Pull request merged**